modules/auxiliary/scanner/oracle/tnspoison_checker.rb

```diff
@@ -42,8 +42,19 @@ def run_host(ip)
       send_packet = tns_packet("(CONNECT_DATA=(COMMAND=service_register_NSGR))")
       sock.put(send_packet)
       packet = sock.read(100)
+      hex_packet = Rex::Text.to_hex(packet, prefix = ':')
+      split_hex = hex_packet.split(":")
       find_packet = /\(ERROR_STACK=\(ERROR=/ === packet
-      find_packet == true ? print_error("#{ip}:#{rport} is not vulnerable ") : print_good("#{ip}:#{rport} is vulnerable")
+      #find_packet == true ? print_error("#{ip}:#{rport} is not vulnerable ") : print_good("#{ip}:#{rport} is vulnerable")
+      if find_packet == true
```

Comment:
i want to make sure i understand the 'is vulnerable' logic.  i read the logic this way:
1) if packet type = `02`, then 'is vulnerable'
2) if packet type = `04` OR packet says 'error', then 'not vulnerable'
3) if i get any other packet type, then 'might be vulnerable'

can you pls confirm 3?  for example, when i look at packet type = `9 ABORT`, i want to make sure you still consider this 'might be vulnerable' and not 'is not vulnerable'.  thanks.